MCDataProducts/src/classes_def.xml

```diff
@@ -95,9 +95,6 @@ include="Math/Vector3D.h, Math/Vector4D.h, CLHEP/Vector/LorentzVector.h, CLHEP/V
 <class name="mu2e::GenElHitData"/>
 <class name="std::pair<unsigned long,mu2e::GenElHitData>"/>
 <class name="std::map<unsigned long,mu2e::GenElHitData>"/>
-<class name="mu2e::VisibleGenElTrack"/>
-<class name="mu2e::VisibleGenElTrackCollection" persistent="false"/>
-<class name="art::Wrapper<mu2e::VisibleGenElTrackCollection>"/>
```

Comment:
> TrackCaloMatching/src/TrkExtrapol_module.cc Do you know if these are needed for other repos or should we just remove them. I can do it as a followon PR in order to keep this one clean.
> its also referenced in classes.h Github says VisibleGenElTrack isn't used anywhere in the Mu2e organization repository besides these lines. I'll remove it. There are some knock-on effects, I'll deal with those too.

Thanks for looking after this.

> ------------------------------ In TrkExt/src/TrkExt_module.cc <[#566 (comment)](https://github.com/Mu2e/Offline/pull/566#discussion_r694145881)>: > @@ -715,7 +695,6 @@ namespace mu2e { krep.getAllCovs(_trkl0, xxcov0, ppcov0, xpcov0); krep.getAllCovs(_trkl1, xxcov1, ppcov1, xpcov1); - covstart_[0][0] = xxcov0[0][0]; covstart_[0][0] = xxcov0[0][0]; Is there a missing line here for the[1][0] element?
> good catch! It looks like this was a spurious line. This module has not been touched in 6 years, it needs major revisions before it can be useful again. Its value is primarily as a reference for future work. For now I suggest to keep it. 

I agree we should keep it for future reference.  There was definitely a spurious duplicate line - but I still think one line is missing, unless the code is assuming a symmetric matrix and only using one of the off-diagonals.

Reply:
> I agree we should keep it for future reference. 

In previous discussion, we had agreed to delete obsolete code, and if it were needed as a reference, dig it out from history [as here](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#Finding_deleted_files)

>but I still think one line is missing, unless the code is assuming a symmetric matrix and only using one of the off-diagonals.

a few lines below it look like the covariance is symmeterized